dashboard/app/models/pd/payment_term.rb

```diff
@@ -62,4 +65,12 @@ def self.for_workshop(workshop)
 
     found_payment_terms[0]
   end
+
+  private
+
+  def sufficient_contract_terms
+    unless per_attendee_payment? || fixed_payment
+      errors.add(:terms, 'Must have either per attendee payment or fixed payment')
+    end
+  end
```

Comment:
this looks like an outdated change from https://github.com/code-dot-org/code-dot-org/pull/14442